eve_sqlalchemy/config/fieldconfig.py

```diff
@@ -56,11 +56,16 @@ def _render(self):
     def _get_field_nullable(self):
         return getattr(self._sqla_column, 'nullable', True)
 
+    def has_server_default(self):
+        return bool(getattr(self._sqla_column, 'server_default'))
```

Comment:
Sorry I did not notice that before: This one should have a leading underscore, as it's entirely internal: `_has_server_default`.

Reply:
ya thought so first time. all others were private. change coming up.